### PR TITLE
820

### PR DESCRIPTION
**Pull request by jospoortvliet**
wip preparation for 8.2.0

TODO:

- [x] put feature text in eight-two page
- [x] create screenshots (WIP)
- [x] update release channels page
- [x] update features page
- [x] update press page
- [ ] changelog (separate)
- [x] include videos
- [ ] update strings.php (will do day-before-release) (separate)
- [ ] update promo banners (will do day-before-release) (separate)

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Hey @phil-davis any chance I could convince you to go over the eight-two page and catch some of the grammar/spelling problems with those attentive eyes of yours? Would rock if the 8.2 feature page is fault-free from the start (would also be a first, hehe)!

File: page-eight-two.php [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+			<div class="col-md-4 featureblock">
+				<i class="icon-desktop"></i>
+				<h3 class="ft-title">Improved Design</h3>
+				<p class="featuredesc">This release introduces significant design improvements including new icons and a sidebar giving quick access to file information.</p>
+			</div>
+
+			<div class="col-md-4 featureblock">
+				<i class="icon-picture-o"></i>
+				<h3 class="ft-title">Pictures Becomes Gallery</h3>
+				<p class="featuredesc">The Gallery app replaces the Pictures with a much nicer user experience introducing image sorting, image zoom, easy switching to the gallery and more.</p>
```
Comment (phil-davis) [inline page-eight-two.php]: Call it the old app "the Pictures app" - consistent with what you already did further down.

File: page-eight-two.php [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+				<h3 class="featuretitle">Pictures Becomes Gallery</h3></br>
+				<p class="featuredesc">The Gallery app replaces the Pictures app for browsing and viewing images. Supporting a wider range of image formats, better performance and new features, Gallery presents a significant enhancement. You can now sort your images, zoom in and out while viewing, and you can configure many specific settings in a <a href="https://github.com/owncloud/gallery/wiki/Gallery-configuration">simple text file</a>.</p>
+			</div>
+
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="documentation" src="<?php bloginfo('template_directory'); ?>/assets/img/features/tips.png" />
+				<h3 class="featuretitle">New Sidebar</h3></br>
+				<p class="featuredesc">A new sidebar in the Files app makes it easy to view details on a file without cluttering the interface. A larger preview with meta data, file activity, sharing state and older versions all reside in the new sidebar. In the future, ownCloud apps can <a href="<?php echo $DOCUMENTATION_DEVELOPER; ?>app">add further functionality</a> through panels in the sidebar.</p>
```
Comment (phil-davis) [inline page-eight-two.php]: Maybe
s/all reside/resides/
It is "A larger preview" that "resides".

File: page-eight-two.php [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="security" src="<?php bloginfo('template_directory'); ?>/assets/img/features/security.png" />
+				<h3 class="featuretitle">Improved Mimetype Handling</h3></br>
+				<p class="featuredesc">Administrators can now can define custom mimetype mappings and aliases. The occ tool has been updated to allow scanning for new mimetypes and update the database to ensure files are properly identified.</p>
+			</div>
+		</div>
+
+
+		<div class="row featurerow">
+                        <div class="col-md-6 featureblock">
```
Comment (phil-davis) [inline page-eight-two.php]: This div has spaces before it - the rest of the lines are indented by tabs.

File: page-eight-two.php [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+				<li>$.get and $.post cannot be used to execute remote JavaScript anymore.</li>
+				<li>The CSRF token is not added to cross-domain requests initiated by jQuery anymore and is displayed differently on each request to mitigate BREACH alike attacks.</li>
+				<li>Referers are not sent anymore to prevent leaking of potential sensitive information within the URL such as the filename.</li>
+				<li>Autoloader is now only loading files from folders that are explicitly whitelisted and only allows loading files from enabled apps.</li>
+				<li>Password reset tokens are now expiring after 12 hours or if a user logged-in again.</li>
+				</ul>
+				Find <a href="https://github.com/owncloud/core/wiki/ownCloud-8.2-Features#security">more changes and details here</a>.
+				</p>
+			</div>
+                        <div class="col-md-6 featureblock">
```
Comment (phil-davis) [inline page-eight-two.php]: This div has spaces before it - the rest of the lines are indented by tabs.

File: page-eight-two.php [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+				<li>Made temporary directory configurable in config.php.</li>
+				<li>Unavailable external storages are not rechecked every request, their failed state is saved and they are tried again after 10 minutes.</li>
+				<li>The upgrade page now states which apps are being upgraded.</li>
+				<li>Downgrades are now prevented as they are unsupported and dangerous.</li>
+				<li>The admin page now warns for PHP versions which are End of Life so admins can prepare to upgrade and avoid security problems which come with unsupported PHP versions.</li>
+				<li>New occ commands to simplify logging configuration and commands to get/set system and app configuration values.</li>
+				</ul>
+				</p>
+			</div>
+                </div>
```
Comment (phil-davis) [inline page-eight-two.php]: and this div end has spaces.

File: page-eight-two.php [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+			<div class="col-md-3"><a href="<?php echo get_template_directory_uri(); ?>/assets/img/screenshots/server/eight-one/LDAP-users-tab.png" title="LDAP wizzard users page" target="_blank"><img src="<?php echo get_template_directory_uri(); ?>/assets/img/screenshots/server/eight-one/LDAP-users-tab-wee.png" style="width: 100%" alt="..." class="img-thumbnail"></a></div>
+			<div class="col-md-3"><a href="<?php echo get_template_directory_uri(); ?>/assets/img/screenshots/server/eight-one/LDAP-login-tab.png" title="LDAP wizzard login page" target="_blank"><img src="<?php echo get_template_directory_uri(); ?>/assets/img/screenshots/server/eight-one/LDAP-login-tab-wee.png" style="width: 100%" alt="..." class="img-thumbnail"></a></div>
+			<div class="col-md-3"><a href="<?php echo get_template_directory_uri(); ?>/assets/img/screenshots/server/eight-one/LDAP-group-tab.png" title="LDAP wizzard group page" target="_blank"><img src="<?php echo get_template_directory_uri(); ?>/assets/img/screenshots/server/eight-one/LDAP-group-tab-wee.png" style="width: 100%" alt="..." class="img-thumbnail"></a></div>
```
Comment (phil-davis) [inline page-eight-two.php]: s/wizzard/wizard/
4 times.

File: page-eight-two.php [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+				<p class="featuredesc">External storage backends are now built upon a 'BackendService' class, decoupling authentication from data handling. This reduces complexity in each external storage backend and makes writing new backends easier. It also makes the existing backends more flexible and capable of dealing with a wider range of authentication mechanisms than just 'username-password'.</p>
+			</div>
+
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="ldap" src="<?php bloginfo('template_directory'); ?>/assets/img/features/avatar.png" />
+				<h3 class="featuretitle">Capabilities and Query Building</h3></br>
+				<p class="featuredesc">A major change is the addition of a capabilities manager which allow apps to expose capabilities in a more generic way. App authors now also have access to a public API for the doctrine query builder.</p>
+			</div>
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="encryption" src="<?php bloginfo('template_directory'); ?>/assets/img/features/encryption.png" />
```
Comment (phil-davis) [inline page-eight-two.php]: Is this the intended thumbnail image here? "encryption" but the text below is more of a general topic.

File: page-eight-two.php [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+				<h3 class="featuretitle">New and Expanded Functions for Apps</h3></br>
+				<p class="featuredesc">As always, there have been many changes to the ownCloud API. For example, applications can now send notifications to end users and mimetype.js makes it possible to request a mimetype icon from JS. To further aid developers, the command 'occ app-check-code'  will notify about deprecated methods and classes.</p>
+			</div>
+		</div>
+
+               <div class="row featurerow">
+			<div class="col-md-4 featureblock">
+				<img class="thumbnail" alt="external_storage" src="<?php bloginfo('template_directory'); ?>/assets/img/features/external.png" />
```
Comment (phil-davis) [inline page-eight-two.php]: and should this thumbnail be external_storage? (which was already use above)

File: page-eight-two.php [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+				<h3 class="featuretitle">Improved External API</h3></br>
+				<p class="featuredesc">The OCS Share API now returns a URL when fetching shares, allows setting expire date on new shares directly and returns permissions and expire date on creation.</p>
+			</div>
+                </div>
+
+		<p>See all features of ownCloud on the <a href="/features">Features page</a>.</p>
+		<h2>Get involved!</h2>
+		<p><a target="_blank" href="/promote/" class="btn btn-lg btn-default"> <i class="icon-bullhorn"></i> Discuss ownCloud with others</a></p>
+		<p><a target="_blank" href="/contribute/" class="btn btn-lg btn-default"> <i class="icon-code"></i> join the ownCloud community</a></p>
```
Comment (phil-davis) [inline page-eight-two.php]: This is standard code on many pages that I noticed the other day. I wonder why "join" is with a small "j"?
"Discuss" starts with a capital letter (above) and "ownCloud" is the correct way it should be written below.
But changing "join" to "Join" is quite a global thing in lots of files.

**Comment by phil-davis** [comment]
Enough comments from me. Since you asked, I was being very picky. But actually it was mostly in good shape.

**Comment by jospoortvliet** [comment]
@phil-davis you rock, dude. Going to make the changes in my next commits ;-)

File: page-eight-two.php (hunk 9 above)
Comment (jospoortvliet) [inline page-eight-two.php]: probably a copy-paste issue... fixed it here but I don't have a nice setup to grep through all files to fix it and... too lazy for now. ;-)

File: page-press.php [hunk 10]
```diff
@@ -1,22 +1,24 @@
...
 <h2>Current Release Information <small> ownCloud Server <?php echo $VERSIONS_SERVER_MAJOR_STABLE; ?></small></h2>
-<!-- <a target="_blank" href="https://owncloud.org/blog/owncloud-server-8-0-released" class="btn btn-lg btn-default">Announcement</a> -->
-<a href="/features" class="btn btn-lg btn-default">Key Features of ownCloud</a>
-<a href="/install" class="btn btn-lg btn-default">Download ownCloud Server <?php echo $VERSIONS_SERVER_MAJOR_STABLE; ?></a>
+<a target="_blank" href="https://owncloud.org/blog/owncloud-server-8-2-brings-renewed-user-interface-and-more-control" class="btn btn-lg btn-default">Announcement</a>
+<a href="/features" class="btn btn-lg btn-default">Overview of new features</a>
 <h2>Latest News and Annoucements</h2>
 <?php wp_get_archives( array( 'type' => 'postbypost', 'limit' => 5, 'format' => 'html' ) ); ?>
 <h2>ownCloud Resources</h2>
+<p>Find an overview of ownCloud history <a href="/history">on our history page</a> and a overview with <a href="/faq">frequently asked questions in our FAQ</a>.</p>
```
Comment (phil-davis) [inline page-press.php]: 2nd mention of "a overview" should be "an overview"

**Comment by jospoortvliet** [comment]
Thanks, fixed!

File: page-eight-two.php [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+				<p class="featuredesc">Administrators can now can define custom mimetype mappings and aliases. The occ tool has been updated to allow scanning for new mimetypes and update the database to ensure files are properly identified.</p>
+			</div>
+		</div>
+
+
+		<div class="row featurerow">
+			<div class="col-md-6 featureblock">
+				<img class="thumbnail" alt="security" src="<?php bloginfo('template_directory'); ?>/assets/img/features/security.png" />
+				<h3 class="featuretitle">Security Improvements</h3></br>
+				<p>ownCloud Server 8.1 has received a significant amount of security hardening, including the following:</br>
```
Comment (LukasReschke) [inline page-eight-two.php]: I guess you mean to refer to 8.2 here?

**Comment by jospoortvliet** [comment]
Ok, moving this to master for final testing and release preparation.

**Pull request merged**